breakout_3_comments.md

```diff
@@ -0,0 +1,16 @@
+# Reviewer Comments
+## Reviewer 1
+### General comments
+I thought this figure looked very good before? I'm not sure what silly reviewer wanted changes done to it, but it doesn't look good anymore!The colors are absurd! I have listed my comments on what to revert or change.
+
+### Specific comments
+- Revert the changes done to the peaches color to its original color. **_Refer to script 1_**
+- Revert the changes done to the leafs color to its original color. **_Refer to script 2_**
+- The text for 'Virtual 2020' is to large and overlapping the line. Reduce the text size to 16. **_Refer to script 5_**
+
+** Hint: You can find the commit that the changes were made and checkout to that commit, then re-commit this for two of these comments.
```

Comment:
are you going to provide any code examples on how to actually do this in git? This handout should walk the user through the process of reverting changes, not just telling them to do so. It's a tutorial, not a quiz.

Reply:
We used to have that in the gitcheat sheet, and this should be addressed in your presentation. 

Breakout 3 is not the place to address this. This should be told to them much more in advance.

Reply:
I thought this was addressed during breakout 1, it's not. So i guess we do it here.

Reply:
> I hate the solutions in the script files.

🙄 

> If you want to put them there I'll let you. But if you don't want them copying and pasting like you said in another comment, then they shouldn't be there. The changes are so simple, if they can't figure it out they won't be able to figure out git.

Yep, no solution files. It was simpler before and allows for easy reference because again, the point of the tutorial isn't how to debug or see if they can figure it out. We need this tutorial to be as user-friendly as possible.

> The markdown had information on checking out to a specific branch, and I'm not sure where it went. I don't think it should be addressed here how to checkout to specific commits but should be done earlier.
>
> This checkout method is going to be a little odd. Because they 'll need to go into their branch and checkout that branch, then pull request again.

We can talk tomorrow about what this breakout room is/isn't supposed to be about.